clients/pricing-client/src/openapi.d.ts

```diff
@@ -602,26 +699,163 @@ declare namespace Components {
              * The taxes applied to the price item.
              */
             recurrences?: (/* An amount associated with a specific recurrence. */ RecurrenceAmountDto)[];
-            _product?: /**
-             * The product entity
+            /**
+             * The snapshot of the product linked to the price item.
              * example:
              * {
              *   "$ref": "#/components/examples/product"
              * }
              */
-            Product;
+            _product?: {
```

Comment:
I think I messed up the types. I've updated the openapi.yaml with some descriptions and looks like the types became expanded like this... need to fix it. i will update this pr